new/rfcXXX-ruby_data_bag_items.md

```diff
@@ -0,0 +1,131 @@
+---
+RFC: unassigned
+Author: Eric Krupnik <ekrupnik@copyright.com>
+Status: Draft
+Type: Standards Track
+---
+# Ruby data bag items
+
+Currently, Chef only supports JSON data bag items, and not Ruby. Chef **does**
+support having other configuration-based files (such as `environments` and `roles`)
+be written in either JSON or Ruby. JSON does not allow comments, so it becomes
```

Comment:
@lamont-granquist I've submitted [this pull request (#432)](https://github.com/opscode/chef-docs/pull/432) against the [chef-docs](https://github.com/opscode/chef-docs) repository to document comment support in data bag items.